pkg/deploy/server/server_configmap.go

```diff
@@ -76,6 +77,7 @@ type CheConfigMap struct {
 	DevfileRegistryUrl                     string `json:"CHE_WORKSPACE_DEVFILE__REGISTRY__URL,omitempty"`
 	DevfileRegistryInternalUrl             string `json:"CHE_WORKSPACE_DEVFILE__REGISTRY__INTERNAL__URL,omitempty"`
 	WebSocketEndpointMinor                 string `json:"CHE_WEBSOCKET_ENDPOINT__MINOR"`
+	WebSocketInternalEndpointMinir         string `json:"CHE_WEBSOCKET_INTERNAL_ENDPOINT__MINOR"`
```

Comment:
```suggestion
	WebSocketInternalEndpointMinor         string `json:"CHE_WEBSOCKET_INTERNAL_ENDPOINT__MINOR"`
```

Reply:
I talked with platform team. They said websocket minor feature was removed one year ago. It's no make sense to leave it in the code, so I removed it.